### ISSUE TITLE
Lexer for bibtex

### ISSUE THREAD

**Issue by nkralpy** [geany/geany#1336]
Hi,

Is it possible to add the lexer for bibtex?

https://sourceforge.net/u/prantl/scintilla/ci/072efce68a054d17f02117feb255eed0c999dc7f/tree/lexers/LexBibTeX.cxx

thanks

**Comment by codebrainz** [comment]
Is this the same as #1297 ?

**Comment by nkralpy** [comment]
Yes, it is. I tried to do it. But couldn't succeed. That's why I raised a request.
